homeassistant/components/device_tracker/__init__.py

```diff
@@ -103,7 +103,7 @@ def see(hass: HomeAssistantType, mac: str=None, dev_id: str=None,
              (ATTR_GPS_ACCURACY, gps_accuracy),
              (ATTR_BATTERY, battery)) if value is not None}
     if attributes:
-        for key, value in attributes:
+        for key, value in attributes.items():
```

Comment:
You should probably only pass the attributes on here `data[ATTR_ATTRIBUTES] = attributes` since this must fall through to `seen` [here](https://github.com/Bart274/home-assistant/blob/e38ab1b2efec72aa221fac4ff921d6b43ed19e3e/homeassistant/components/device_tracker/__init__.py#L363)